src/DataSets/DataSets.component.js

```diff
@@ -74,6 +74,10 @@ const DataSets = React.createClass({
         };
     },
 
+    tr(text) {
```

Comment:
Not used here, but if you ever need it, `getTranslation` has a second argument `variables = {}`, used as a namespace for string interpolation. In i18n files you then write: `key=my name is $$name$$`.

Reply:
Neat, I didn't know. I'm adding it to the function signature.

I'm happy you didn't think it was too silly to add this tr() function, btw. :)